python-sdk/docs/astro/files/operators/get_file_list.rst

```diff
@@ -10,9 +10,18 @@ You can use ``get_file_list`` to retrieve a list of available files based on a s
 
 The supported filesystems are :ref:`file_location`
 
+.. warning::
+    Fetching a lots of files using this method can lead to overload XCOM and also can create lots of parallel tasks if using in dynamic task map expand method.
```

Comment:
```suggestion
    Fetching a lot of files using this method can lead to overloaded XCOM. This can create lot of parallel tasks when used in dynamic task map ``expand`` method.
```